test/lib/JtregJextract.java

```diff
@@ -71,16 +71,6 @@ private String[] processArgs(String... args) {
                 break;
             }
 
-            if ("-libpath".equals(opt)) {
```

Comment:
I've removed this, as I think this was a questionable option. I'm not exactly sure what problems this was meant to address, but the only test using this has been rewritten to use other options instead.